packages/cli/src/api/help.ts

```diff
@@ -15,7 +16,26 @@
  * ...
  * (use "npm run compile" to compile catalogs for production)
  */
-export function helpRun(command) {
+import path from "path"
+
+export function helpRun(command: string) {
+  let findRootPkgJson: Record<string, unknown>;
+  try {
+    findRootPkgJson = require(path.resolve(
+      path.join(process.cwd(), "package.json")
+    ))
+  } catch (error) {}
+
+  if (findRootPkgJson?.scripts) {
+    const res = Object
+      .entries(findRootPkgJson.scripts)
+      .find(([_, value]) => value.includes(`lingui ${command}`))
+
+    if (res[0]) {
```

Comment:
Isn't `res == null` when it doesn't find a matching command?

```suggestion
    if (res != null) {
```

Reply:
Ahh, right damn it, i'll change it asap

Reply:
I don't why I tought that entries was returning an empty array if not found, changes applied! :shipit: